src/GovUk.Education.ExploreEducationStatistics.Common/Extensions/StringExtensions.cs

```diff
@@ -1,10 +1,43 @@
+using System.Collections.Generic;
 using System.Globalization;
+using System.IO;
+using System.Linq;
 using System.Text.RegularExpressions;
+using AngleSharp.Text;
 
 namespace GovUk.Education.ExploreEducationStatistics.Common.Extensions
 {
     public static class StringExtensions
     {
+        /// <summary>
+        /// Convert a string into an enumerable of its constituent lines.
+        /// </summary>
+        public static IEnumerable<string> ToLines(this string value)
+        {
+            var reader = new StringReader(value);
```

Comment:
Not familiar with StringReaders, but do you need to close them after use?  Or use a "using" block at all?  Quite possibly not, but thought I'd ask!

Reply:
Good point! I _don't think_ `StringReader` needs to close necessarily as it's not holding onto any IO-bound resource, but apparently, it's just 'good form' to do so anyway.